.github/workflows/build.yaml

```diff
@@ -40,11 +46,20 @@ jobs:
       key: ${{ secrets.PRIVATE_KEY }}
   data:
     needs: [repos]
+    strategy:
+      fail-fast: false
+      matrix:
+        site: ${{ fromJSON(needs.repos.outputs.repos) }}
     uses: hubverse-org/hub-dashboard-control-room/.github/workflows/generate-data.yaml@main
     with:
+      owner: '${{ matrix.site.owner }}'
+      name: '${{ matrix.site.name }}'
       repos: '${{ needs.repos.outputs.repos }}'
       slug: '${{ needs.repos.outputs.slug }}'
       email: '${{ needs.repos.outputs.email }}'
+      regenerate: ${{ github.event.client_payload.regenerate || inputs.regenerate || false }}
+      build: ${{ github.event.client_payload.build || 'both' }}
+      publish: ${{ inputs.publish || github.event_name != 'workflow_dispatch' }}
```

Comment:
A note about these variables:

 - `github.event.client_payload` comes from a call from the app (either a push or issue comment). If this wasn't called from the app, this value ends up being an empty string, which evaluates to `false`
 - `inputs` is from the `workflow_dispatch` and has the default defined above.